sdk/spring/azure-spring-boot/src/main/java/com/azure/spring/autoconfigure/aad/AADAuthenticationProperties.java

```diff
@@ -230,6 +240,11 @@ public void validateUserGroupProperties() {
             throw new IllegalArgumentException("One of the User Group Properties must be populated. "
                 + "Please populate azure.activedirectory.user-group.allowed-groups");
         }
+        if (!DEFAULT_GROUP_RELATIONSHIP.equalsIgnoreCase(groupRelationship)
+            && !"transitive".equalsIgnoreCase(groupRelationship)) {
```

Comment:
Why don't we have a static variable for this one?

Reply:
Will add another one.